hack/validate-acceptable-bundles.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+# Copyright 2023 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Use the Enterprise Contract to validate the Tekton pipeline bundles being added to
+# data/acceptable_tekton_bundles.yml. The list of bundles to verify is a computed by
+# diff'ing against the list of bundles in origin/main. If there's no difference, this
+# script succeeds.
+# Usage:
+#   hack/validate-acceptable-bundles.sh
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+bundles_file='data/acceptable_tekton_bundles.yml'
+
+function list_pipeline_bundles() {
+    < "${1}" yq \
+        '."pipeline-bundles" | to_entries | .[] | .key + "@" + .value[].digest' | \
+        sort -u
```

Comment:
(nitpick, for fun) you could use [`unique`](https://mikefarah.gitbook.io/yq/operators/unique)

Something like:

```suggestion
    < "${1}" yq \
        '[."pipeline-bundles" | to_entries | .[] | .key + "@" + .value[].digest] | unique | .[]'
```

Reply:
Yes! Also need `sort`